backend/src/routes/api/gpu/gpuUtils.ts

```diff
@@ -79,3 +89,36 @@ export const getGPUData = async (
     httpsRequest.end();
   });
 };
+
+const getGPUScaling = async (fastify: KubeFastifyInstance): Promise<gpuScale[]> => {
+  const scalingList: gpuScale[] = [];
+  const autoscalerList = (
+    await fastify.kube.customObjectsApi.listNamespacedCustomObject(
+      'autoscaling.openshift.io',
+      'v1beta1',
+      'openshift-machine-api',
+      'machineautoscalers',
+    )
+  ).body as MachineAutoscalerList;
+  for (let i = 0; i < autoscalerList.items.length; i++) {
+    const machineSetName = autoscalerList.items[i].spec.scaleTargetRef.name; //also gives info about kind and apiversion if needed in the future
+    const machineSet = (
+      await fastify.kube.customObjectsApi.getNamespacedCustomObject(
```

Comment:
This await in a loop is going to be awfully slow.  Can we add all the promises to an array and do a Promises.all.  I mean it would even be faster to get all the machinesets with a limit and just filter out the ones that don't autoscale.

Reply:
Ok, added a loop that works with Promise.all